src/trigger.py

```diff
@@ -34,8 +34,10 @@ def _log_revision(self, message, build_config, head_commit):
 
     def _run_trigger(self, build_config, force, timeout):
         head_commit = kernelci.build.get_branch_head(build_config)
+        whoami = self._api.whoami()
```

Comment:
This could be one only once when the service starts, there's already a class to store the `whoami` data.  No need to keep asking the API if I'm still user, unless an identity crisis is likely to be triggered rather than a new kernel revision ;)

If something as critical as the username of the user running the pipeline changes, I would expect a full synchronous redeploy.

Reply:
Ok, i will update code, agree as it might significantly increase load on API if it will poll whoami each time